runtime/src/lib.rs

```diff
@@ -642,6 +630,49 @@ impl pallet_session::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub IntergalacticAccounts: Vec<AccountId> = vec![
+		hex_literal::hex!["0abad795adcb5dee45d29528005b1f78d55fc170844babde88df84016c6cd14d"].into(),
```

Comment:
how you can use vesting in dev environment when this account is only allowed one, sudo?

Reply:
hmm, i dont see this filter applied anywhere

Reply:
not sure if sudo will do, this is not filtering origin of funds but the transaction, so only that specific wallet can call it

Reply:
I think this is going to be transferred from treasury (like a regular claim) so RootOrTreasury might be the best candidate for origin

Reply:
depends, i assume that hydra dx forked balances will be minted, same as the lbp claims

Reply:
i'll rephrase question: this filter means that you can vest tokens only by call from intergalacic address?

Reply:
yes that is correct. not only that! moreover the balance is transferred from the callers account so the caller has to have enough balances. For now, I changed the origin to RootOrTreasury.